external_search.py

```diff
@@ -2331,6 +2332,53 @@ def __init__(self, collections=None, media=None, languages=None,
         else:
             self.scoring_functions = []
 
+    @property
+    def audiences(self):
+        """Return the appropriate audiences for this query.
+
+        This will be whatever audiences were provided, but it will
+        probably also include the 'All Ages' audience, and it may exclude
+        the 'Research' audience.
```

Comment:
Also this bit of the comment is out of date; we no longer exclude 'Research' here.